evm/src/main/java/org/hyperledger/besu/evm/operation/AbstractCreateOperation.java

```diff
@@ -74,6 +78,10 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
 
       final long inputOffset = clampedToLong(frame.getStackItem(1));
       final long inputSize = clampedToLong(frame.getStackItem(2));
+      if (inputSize > maxInitcodeSize) {
+        frame.popStackItems(getStackItemsConsumed());
+        return new OperationResult(cost, ExceptionalHaltReason.CODE_TOO_LARGE);
+      }
```

Comment:
Does this warrant a test at the Operation level?

Reply:
Tests were there, but testing the wrong thing.  Fixed.